apps/src/StudioApp.js

```diff
@@ -2443,7 +2443,7 @@ StudioApp.prototype.handleEditCode_ = function(config) {
         this.editor.enablePalette(!this.editor.session.paletteEnabled);
         showToolboxHeader.style.display = this.editor.session.paletteEnabled
           ? 'none'
-          : 'inline-block';
+          : 'flex';
```

Comment:
more evidence that there is too much is StudioApp!

Reply:
ugh yes, banged my head against a wall for a while before I found this little nugget 🤦

Reply:
Could you explain why you had to change from 'inline-block' to 'flex'?

Reply:
mmm I made the element with this ID (`showToolboxHeader`) a flex container in CodeWorkspace.jsx, and this code changes this attribute when the toolbox is shown/hidden in Gamelab/Applab (not sure why this code is here, to Molly's point). So, this is for consistency with what's happening in CodeWorkspace.jsx.

I don't totally remember why I changed that container to a flexbox -- generally, I like using flex containers when you have 3 elements and want a left, centered, and right element (which is what we want here).

Reply:
Thanks so much! Where would you expect to see this snippet of code if not in `StudioApp.js`? `CodeWorkspace.jsx`?

Reply:
I think so? I'm sure there's some reason it's there, but it's unusual that we'd have such specific CSS (only applicable to a small part Gamelab and Applab) in here.